src/components/dialog-full-screen/dialog-full-screen.component.js

```diff
@@ -74,21 +76,32 @@ class DialogFullScreen extends Modal {
   /**
    * Overrides the original function to disable the document's scroll.
    */
-  get onOpening() {
-    this.document.documentElement.style.overflow = 'hidden';
+  onOpening() {
+    this.focusTrap = new FocusTrap(this._dialog);
+    this.focusTrap.setFocusTrap();
 
-    return this.document.documentElement;
+    this.document.documentElement.style.overflow = 'hidden';
```

Comment:
We should store the original value of `overflow` in case someone has already changed it.